telcell/geography.py

```diff
@@ -0,0 +1,128 @@
+    radians.
+    """
+
+    def __init__(self, degrees: float = None, radians: float = None):
+        assert (
+            degrees is None or radians is None
+        ), "angle requires either degrees or radians; both provided"
+        if degrees is not None:
+            self._degrees = degrees
+        elif radians is not None:
+            self._degrees = geopy.units.degrees(radians=radians)
+        else:
+            raise ValueError("angle requires either degrees or radians; none provided")
+
+    def __eq__(self, other):
+        if isinstance(other, Angle):
+            return self.degrees == other.degrees
+        else:
+            return self.degrees == other
+
+    def __gt__(self, other):
+        if isinstance(other, Angle):
+            return self.degrees > other.degrees
+        else:
+            return self.degrees > other
+
+    def __ge__(self, other):
+        if isinstance(other, Angle):
+            return self.degrees >= other.degrees
+        else:
+            return self.degrees >= other
+
+    def __lt__(self, other):
+        if isinstance(other, Angle):
+            return self.degrees < other.degrees
+        else:
+            return self.degrees < other
+
+    def __le__(self, other):
+        if isinstance(other, Angle):
+            return self.degrees <= other.degrees
+        else:
+            return self.degrees <= other
+
+    def __ne__(self, other):
+        if isinstance(other, Angle):
+            return self.degrees != other.degrees
+        else:
+            return self.degrees != other
+
+    def __mul__(self, other):
+        return Angle(degrees=self.degrees * other)
+
+    def __truediv__(self, other):
+        return Angle(degrees=self.degrees / other)
+
+    def __add__(self, other):
+        if isinstance(other, Angle):
+            return Angle(degrees=self.degrees + other.degrees)
+        else:
+            raise TypeError(
+                f"unsupported operand types for +: {type(self)} and {type(other)}"
+            )
+
+    def __sub__(self, other):
+        if isinstance(other, Angle):
+            return Angle(degrees=self.degrees - other.degrees)
+        else:
+            raise TypeError(
+                f"unsupported operand types for -: {type(self)} and {type(other)}"
+            )
+
+    def __repr__(self):
+        return f"Angle(degrees={self._degrees})"
+
+    def isnan(self):
+        return math.isnan(self.degrees)
+
+    @property
+    def degrees(self):
+        return self._degrees
+
+    @property
+    def radians(self):
+        return geopy.units.radians(degrees=self._degrees)
+
+
+def normalize_angle(angle: Angle) -> Angle:
+    """
+    Normalizes an angle to be in the range [-180,180> degrees.
+    """
+    normalized = (angle.degrees + 180) % 360 - 180
+    return Angle(degrees=normalized)
+
+
+def azimuth_deg(coord1: geopy.Point, coord2: geopy.Point) -> float:
```

Comment:
we hebben ook in telcell/data/models al een Point klasse gedefinieerd. Het lijkt me beter om 1 optie te kiezen in de hele repo

Reply:
Eens, dit zou wat mij betreft `geopy.Point` worden; hiervoor een nieuwe PR om monster-PR's te voorkomen